styx-common/src/main/java/com/spotify/styx/model/TriggerResponse.java

```diff
@@ -0,0 +1,42 @@
+/*-
+ * -\-\-
+ * Spotify Styx Common
+ * --
+ * Copyright (C) 2016 - 2023 Spotify AB
+ * --
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -/-/-
+ */
+
+package com.spotify.styx.model;
+
+import io.norberg.automatter.AutoMatter;
+
+@AutoMatter
+public interface TriggerResponse extends TriggerRequest {
+  String triggerId();
+
+  static TriggerResponseBuilder builder() {
+    return new TriggerResponseBuilder();
+  }
+
+  static TriggerResponse of(WorkflowId workflowId, String parameter,
```

Comment:
Maybe a factory method `TriggerResponseBuilder from(TriggerRequest triggerRequest)` could be more useful. A small thing, doesn't matter.

Reply:
I think I would rather not leak the builder, but it's not a strong objection 🤷‍♀️